stable/influxdb/values.yaml

```diff
@@ -23,6 +23,40 @@ persistence:
   accessMode: ReadWriteOnce
   size: 8Gi
 
+## Create default user through Kubernetes job
+## Defaults indicated below
+##
+setDefaultUser:
+  enabled: false
+
+  ## Image of the container used for job
+  ## Default: appropriate/curl:latest
+  ##
+  # image: appropriate/curl:latest
+
+  ## Deadline for job so it does not retry forever.
+  ## Default: activeDeadline: 300
+  ##
+  # activeDeadline: 300
+
+  ## Restart policy for job
+  ## Default: OnFailure
+  # restartPolicy: OnFailure
+
+  # user:
+
+    ## The user name
+    ## Default: "admin"
+    # username: "admin"
+
+    ## User password
+    ## Default: "admin"
```

Comment:
This is no longer correct. The password will default to `{{ randAscii 10 }}`. Please be sure to note this in the `values.yaml` file.